src/calendar/index.js

```diff
@@ -158,6 +165,13 @@ class Calendar extends Component {
     this.updateMonth(this.state.currentMonth.clone().addMonths(count, true));
   }
 
+  intzAccessibilityLabel(state, day) {
```

Comment:
what is `intz`? do you mean `init`?
also, arrow functions.

Reply:
internationalization, i could change with i18nAccesibilityLabel?

Reply:
yes